### ISSUE TITLE
Convert base58 to hex

### ISSUE THREAD

**Issue by fede-rodes** [Ethernautas/renpool#36]
We need to implement the base58 to hex conversion in order to process the node ID given by the darknode CLI.

For instance, when running the darknode cli you get the following output:
```
Congratulations! Your Darknode is deployed.
Join the network by registering your Darknode at https://mainnet.renproject.io/darknode/8MGTMfNPeekDnSrJDqmf5oSVZaK3Vx?action=register&public_key=0x000000077373682d727361000000030100010000010100d0feba4ae65ea9ad771d153419bcc21189d954b6bf75fd5488055cd2641231014f190c0e059a452d301c535e931df33590ec0e18c59341a2766cc885d1dc6e66f5cc65b94522ec944ae4200bd56a30223328b258d50b507dd94b4c4742768f3fec2b815c9c4b0fe26727e82865f6a064fa3ff2443d135d9788095a1c17487fd5c389a491c16b73385d516a303debc3bcccae337a7ec0d89d51ce05262a0c4c1f2178466c85379b8cd4e5cbe1c90a05fb0c1ed3eee2134774b450e7b0b70c792abad55beef919e21a03cb9de4e963a820c2f84421a4559d0b67cfd17c1686ff6f2d1bb07ac2c82cede1cf5f16a57e125a29fef65891715b061606bca1a0eb026b&name=darknode-kovan
```
If I understand correctly the `8MGTMfNPeekDnSrJDqmf5oSVZaK3Vx` is the nodeID which is required when registering the node (together with the publicKey). Now, in order for the register function to work the nodeID needs to be an `address`. Therefore we should convert from base58ToHex.

See how the conversion is done in the command center:
https://github.com/renproject/command-center/blob/9aa947a6fd1ec3459c06809e4fb82915032805a6/src/lib/darknode/darknodeID.ts#L7

Here's the register function of the DarknodeRegistry:
https://github.com/Ethernautas/darknode-sol/blob/master/contracts/DarknodeRegistry/DarknodeRegistry.sol#L232

And here where we should use the encoding:
https://github.com/Ethernautas/renpool/blob/master/tests/ren_pool/test_darknode_registration.py#L6

**Comment by JonathanStefanov** [comment]
Yes exactly I had noticed this too, but this conversation should be made in our front end right? When the node operator will set up the pool they will enter the base58 ID in our front end and we will do the conversion there before calling our smart contract with the right address. Correct?

**Comment by fede-rodes** [comment]
That's correct, we need this in the front end to convert nodeID into hex before calling the `registerDarknode` method of the `RenPool` contract. However, It'd also be nice to have this implemented in python since we could use this functionality in the register darknode tests.

@JonathanStefanov do you mind sharing the url that you got when registering the node plus the nodeID (hex) value you have when opening say link?

**Comment by fede-rodes** [comment]
Here's a nice converter in case it helps: https://www.mobilefish.com/services/cryptocurrency/brainwallet.html#converter
